background/redux-slices/utils/asset-utils.ts

```diff
@@ -352,3 +352,10 @@ export function isUntrustedAsset(asset: AnyAsset | undefined): boolean {
   }
   return false
 }
+
+export function isCustomAsset(asset: AnyAsset | undefined): boolean {
```

Comment:
I would assume the asset will be always present when passed to this function - expecting undefined may be more convenient but may also hide some bugs
```suggestion
export function isCustomAsset(asset: AnyAsset): boolean {
```